doc/reference/configuration/configuration_reference.rst

```diff
@@ -3824,6 +3824,219 @@ snapshot.by.*
+
+..  NOTE::
+
+    ``vinyl`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+-   :ref:`vinyl.bloom_fpr <configuration_reference_vinyl_bloom_fpr>`
+-   :ref:`vinyl.cache <configuration_reference_vinyl_cache>`
+-   :ref:`vinyl.defer_deletes <configuration_reference_vinyl_defer_deletes>`
+-   :ref:`vinyl.dir <configuration_reference_vinyl_dir>`
+-   :ref:`vinyl.max_tuple_size <configuration_reference_vinyl_max_tuple_size>`
+-   :ref:`vinyl.memory <configuration_reference_vinyl_memory>`
+-   :ref:`vinyl.page_size <configuration_reference_vinyl_page_size>`
+-   :ref:`vinyl.range_size <configuration_reference_vinyl_>`
+-   :ref:`vinyl.read_threads <configuration_reference_vinyl_read_threads>`
+-   :ref:`vinyl.run_count_per_level <configuration_reference_vinyl_run_count_per_level>`
+-   :ref:`vinyl.run_size_ratio <configuration_reference_vinyl_run_size_ratio>`
+-   :ref:`vinyl.timeout <configuration_reference_vinyl_timeout>`
+-   :ref:`vinyl.write_threads <configuration_reference_vinyl_write_threads>`
+
+..  _configuration_reference_vinyl_bloom_fpr:
+
+..  confval:: vinyl.bloom_fpr
+
+    Bloom filter false positive rate -- the suitable probability of the
+    `bloom filter <https://en.wikipedia.org/wiki/Bloom_filter>`_
+    to give a wrong result.
+    The ``vinyl.bloom_fpr`` setting is a default value for one of the
+    options in the :ref:`Options for space_object:create_index() <box_space-create_index>` chart.
+
+    |
+    | Type: float
+    | Default: 0.05
+    | Environment variable: TT_VINYL_BLOOM_FPR
+
+..  _configuration_reference_vinyl_cache:
+
+..  confval:: vinyl.cache
+
+    The cache size for the vinyl storage engine. The cache can
+    be resized dynamically.
+
+    |
+    | Type: integer
+    | Default: 128 * 1024 * 1024
+    | Environment variable: TT_VINYL_CACHE
+
+..  _configuration_reference_vinyl_defer_deletes:
+
+..  confval:: vinyl.defer_deletes
+
+    Enable the deferred DELETE optimization in vinyl. It was disabled by default
+    since Tarantool version 2.10 to avoid possible performance degradation
+    of secondary index reads.
+
+    |
+    | Type: boolean
+    | Default: false
+    | Environment variable: TT_VINYL_DEFER_DELETES
+
+..  _configuration_reference_vinyl_dir:
+
+..  confval:: vinyl.dir
+
+    A directory where vinyl files or subdirectories will be stored.
+
+    This option may contain a relative file path.
+    In this case, it is interpreted as relative to
+    :ref:`process.work_dir <configuration_reference_process_work_dir>`.
+
+    |
+    | Type: string
+    | Default: var/lib/{{ instance_name }}'
+    | Environment variable: TT_VINYL_DIR
+
+..  _configuration_reference_vinyl_max_tuple_size:
+
+..  confval:: vinyl.max_tuple_size
+
+    Size of the largest allocation unit,
+    for the vinyl storage engine. It can be increased if it
+    is necessary to store large tuples.
+    See also: :ref:`memtx.max_tuple_size <configuration_reference_memtx_max_tuple_size>`.
```

Comment:
1. I would place the 'See also' in its own paragraph.
2. The link is not rendered on the website.